techniques/failures/F111.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Failure of Success Criterion 2.5.3 due to a control with visible label text but no accessible name</title>
```

Comment:
This creates an inconsistency with ACT. Our [Form field has non-empty accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/e086e5/) rule maps to only to 4.1.2. It does not allow failing form fields with empty accessible names under 2.5.3. I can appreciate the argument for putting it here, I'm not sure why it wouldn't primarily be a 4.1.2 issue.

@kengdoj You may want to have a look at this.

Reply:
I don't think WCAG has a concept of something failing "primarily" one SC over another. Maybe this technique should be expanded to fail both 2.5.3 and 4.1.2?

Reply:
Agreed, and I can definitely see an argument for doing both. I've asked Kathy to take it up in ACT.

Reply:
This procedure is a sub-set of things that could fail 4.1.2. I.e. it requires that there is a visible label, and that the text is contained in the accname.
From the TF discussion: We don't think this mis-alignment should hold things up, it is ok to have slightly different failures for different SCs, but we'll give some time to think about this one.

Reply:
would submit this is also an example of a 1.3.1 failure.  i'm not sure it matters if ACT doesn't log all the SCs that could be failed for a particular pattern, so long as it is consistent in what the act checks can find.

Reply:
I'm currently on a hiatus from ACT. Please coordinate with them.